include/internal/libspdm_common_lib.h

```diff
@@ -279,6 +279,25 @@ typedef struct {
     libspdm_large_managed_buffer_t certificate_chain_buffer;
 } libspdm_encap_context_t;
 
+typedef struct {
+    bool chunk_in_use;
+
+#if LIBSPDM_ENABLE_CHUNK_CAP
```

Comment:
I did it this way to avoid a zero sized struct compile error for the member libspdm_chunk_context_t and in libspdm_context_t. And it didn't seem like any other code was being conditionally compiled within the libspdm_context_t. I can wrap the entire struct, but then that would introduce the #if/#endif into libspdm_context_t as well.

Reply:
Yes, I think we can add if/endif around libspdm_chunk_context_t. That is a cleaner solution, if we dont need CHUNK.

Reply:
ok.